homeassistant/components/imap/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
Unfortunately, that is not the case. The upstream dependency matching the following part of this rule:

> The package published to PyPi should be built and published inside a CI pipeline.

It seems like `aioimaplib` only runs tests in their pipelines.

Reply:
I'll set this one to `todo`.